helix-dap/src/client.rs

```diff
@@ -33,31 +33,32 @@ pub struct Client {
     request_counter: AtomicU64,
     connection_type: Option<ConnectionType>,
     starting_request_args: Option<Value>,
+    /// The socket address of the debugger, if using TCP transport.
+    pub socket: Option<SocketAddr>,
     pub caps: Option<DebuggerCapabilities>,
     // thread_id -> frames
     pub stack_frames: HashMap<ThreadId, Vec<StackFrame>>,
     pub thread_states: ThreadStates,
-    pub thread_id: Option<ThreadId>,
+    /// The currently active thread (usually the one we've broken at).
+    pub active_thread_id: Option<ThreadId>,
```

Comment:
Please avoid renaming things, it only makes the diff harder to follow. Adding docs explaining its use is good though

Reply:
Sure, I can revert it. This was to be inline with `active_frame` just below